dashboards-observability/public/components/notebooks/components/__tests__/note_table.test.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fireEvent, render } from '@testing-library/react';
+import { configure, mount, shallow } from 'enzyme';
```

Comment:
Doesn't look like you use mount or shallow anywhere???

Reply:
true, after the code base is stable i'll format all files